bundles/org.openhab.binding.meross/README.md

```diff
@@ -0,0 +1,95 @@
+# Meross Binding
+
+This binding integrates **Meross**&reg; devices
+
+## Supported Things
+
+Supported thing types
+
+- bridge
+- light
+
+|   Meross Name       | Type   | Description         | Supported | Tested| 
+|---------------------|--------|---------------------|-----------|--------|
+| Smart ambient light | msl430 | Smart ambient light | yes       | yes    |
+| Smart plug          | mss210 | Smart plug          | yes       | yes    |
+
+Anyway, the togglex capability may consent to control on/off of the majority of plugs and bulbs
+
+## Discovery
+
+The Discovery service is supported.
+if a refresh of devices is needed, e.g. to fetch new devices please disable  and re-enable the bridge via the interface button 
+
+## Binding Configuration
+
+To utilize the binding you should first create an account via the **Meross**&reg; 
+**Android**&reg; app. 
+Moreover, the devices should be in an online status
+
+## Bridge Configuration
+
+| Name     | Type | Description                                              | Default                    | Required | Advanced |
+|----------|------|----------------------------------------------------------|----------------------------|----------|----------|
+| hostname | text | Meross Hostname or IP address (for Europe located users) | https://iotx-eu.meross.com | yes      | yes      |
+| email    | text | Email of your Meross Account                             | N/A                        | yes      | no       |
+| password | text | Password of your Meross Account                          | N/A                        | yes      | no       |
+
+NOTICE: Due to  **Meross**&reg; security policy please minimize host connections in order to
+avoid TOO MANY TOKENS  (code 1301) error occurs which leads to a  8-10 hours suspension of your account.
+
+
+## Thing Configuration
+
+| Parameter | Type | Description                                             | Default | Required | Thing type id | Advanced |
+|-----------|------|---------------------------------------------------------|---------|----------|---------------|----------|
+| lightName | text | The name of the light as registered to Meross account   | N/A     | yes      | light         | no       |
+
+## Channels
+
+Only togglex channel is supported:
+
+| Channel | Type   | Read/Write | Description                                                  |
+|---------|--------|------------|--------------------------------------------------------------|
+| togglex | Switch | N/A        | Power bulb/plug capability to control bulbs and plugs on/off |
+
+NOTICE: Due to  **Meross**&reg; security policy  please limit  communication to no more than
+150 messages every one hour at the earliest convenience otherwise, the user is emailed  by
+Meross of the limit exceed and if such a behaviour does not change the user's  account will be
+**BANNED**! 
```

Comment:
Whitespace, one line etc
```suggestion
NOTICE: Due to **Meross**&reg; security policy please limit communication to no more than 150 messages every one hour at the earliest convenience otherwise, the user is emailed by Meross of the limit exceed and if such a behaviour does not change the user's account will be **BANNED**! 
```